### ISSUE TITLE
VPAT — Discussion Forum Is Missing Form Labels

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#5041]
An internal accessibility audit found parts of the discussion forum is missing form labels.  That is, some form elements do not have an associated `<label>` tag.  

Form labels help assistive technology to identify or describe a form element to a user.  Alternatively, we can use `aria-label` or `aria-describedby` properties when form labels would introduce unnecessary burden with UI design.
